### PR TITLE
Timers

### PR DESCRIPTION
**Pull request by allaffa**

### CODE REVIEW

File: hydragnn/run_training.py [hunk 1]
```diff
@@ -242,3 +243,5 @@ def _(config: dict):
             model.state_dict(),
             "./logs/" + model_with_config_name + "/" + model_with_config_name + ".pk",
         )
+
+    print_timers(verbosity)
```
Comment (streeve) [inline hydragnn/run_training.py:247]: We should print in `run_prediction` as well

**Review by streeve — CHANGES_REQUESTED** [review]
There are some unresolved comments plus one more new one

File: hydragnn/run_prediction.py [hunk 2]
```diff
@@ -126,3 +127,4 @@ def run_prediction(config: dict):
     ) = test(test_loader, model, config["Verbosity"]["level"])
 
     return error, error_sumofnodes_task, error_rmse_task, true_values, predicted_values
+    print_timers(verbosity)
```
Comment (streeve) [inline hydragnn/run_prediction.py:130]: Before return :)

File: hydragnn/utils/time_utils.py [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+        )
+    ]
+
+    # The statistics are the result of global collective operations, so we only print them once
+    if verbosity >= 1:
+        if world_size > 1:
+            print_distributed(1, "Minimum timers: ")
+            [
+                print_distributed(1, key, " : ", round(value, 2))
+                for key, value in Timer.timers_min.items()
```
Comment (streeve) [inline hydragnn/utils/time_utils.py:110]: Add the same sorting as individual timers

File: hydragnn/run_prediction.py (hunk 2 above)
Comment (allaffa) [reply hydragnn/run_prediction.py:129]: LOL. Thanks :)

**Review by streeve — APPROVED** [review]

**Pull request merged**